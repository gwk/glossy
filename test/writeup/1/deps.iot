{
  'cmd': 'writeup -deps embed.wu',
  'links': {
    'test/writeup/1/html/embed.wu',
    'test/writeup/assets',
  },
}
