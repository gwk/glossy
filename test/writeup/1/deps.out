assets/circle.svg
assets/empty.csv
assets/head.gif
assets/head.jpg
assets/head.png
assets/line.wu
assets/nested.wu
assets/table.csv
assets/text.txt
